firefox-ios/Shared/Supporting Files/ja.lproj/SuggestedSites.strings

```diff
@@ -0,0 +1,3 @@
+/* Title for X (formerly Twitter) tile in the suggested sites section of the homepage. */
+"SuggestedSites.X.Title.v131" = "X (旧 Twitter)";
```

Comment:
Another of these cases @Delphine, where we don't need the (formerly Twitter) part in the translation. 🙏